src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,29 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: $card-width;
+  gap: $card-gap;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-width);
+  }
```

Comment:
You can make a loop to create all media queries. Just create a correctly mapped list to iterate.